.github/ISSUE_TEMPLATE/cherry-pick-request.yml

```diff
@@ -0,0 +1,158 @@
+---
+name: Cherry-Pick Request
+description: Request a cherry-pick to an AMP release.
+labels: ["Type: Release", "Cherry-pick: Beta", "Cherry-pick: Experimental", "Cherry-pick: LTS", "Cherry-pick: Stable"]
+title: "\U0001F338 Cherry-pick request for #ISSUE into #RELEASE"
+body:
+- type: markdown
+  id: header
+  attributes:
+    value: |
+      Thanks for filling out this cherry-pick request.
+      - See AMP's [release schedule](https://github.com/ampproject/amphtml/blob/main/docs/release-schedule.md) to learn about how releases work.
+      - See AMP's [release calendar](https://amp-release-calendar.appspot.com) for information about past releases (e.g. versions, dates, submission times, notes).
+      - See AMP's [code contribution docs](https://github.com/ampproject/amphtml/blob/main/docs/contributing-code.md#Cherry-picks) for an overview of the cherry-pick process.
+      - Use this form to provide the following details:
+        - The bug that necessitates this cherry-pick.
+        - The PR(s) that fix the bug.
+        - The release to which the cherry-pick will be applied.
+        - The channel(s) to which the cherry-pick will be applied.
+- type: input
+  id: issue
+  attributes:
+    label: Issue
+    description: The bug that necessitates this cherry-pick. Remember to update the issue title with this info.
+    placeholder: "e.g. #11111"
+  validations:
+    required: true
+- type: input
+  id: pull_request
+  attributes:
+    label: Pull Request(s)
+    description: The PR(s) that fix the bug. Make sure they are merged and have passing [CI builds](https://app.circleci.com/pipelines/github/ampproject/amphtml?branch=main).
+    placeholder: "e.g. #22222, #33333"
+  validations:
+    required: true
+- type: input
+  id: release_tracker
```

Comment:
This should support multiple values for the use case of CPing into multiple channels